ci/prow/make_config.go

```diff
@@ -924,6 +925,13 @@ func generatePeriodic(title string, repoName string, periodicConfig yaml.MapSlic
 		addEnvToJob(&data.Base, "GOOGLE_APPLICATION_CREDENTIALS", data.Base.ServiceAccount)
 		addEnvToJob(&data.Base, "E2E_CLUSTER_REGION", "us-central1")
 	}
+	if version != "" {
+		// If it's a release version, add env var PULL_BASE_REF as ref name of the base branch.
+		// NOTE:
+		// This serves as a workaround since Prow does not have PULL_BASE_REF set for periodic jobs - https://github.com/kubernetes/test-infra/blob/abcd35c4dbfb0feadd09fc452b533222e3a16b29/prow/jobs.md.
+		// We are checking with Kubernetes test-infra team. If we can actually add this env var for all periodic jobs, we can safely delete it here.
```

Comment:
Replace this explanation with a TODO and a link to an issue, so we document whatever the outcome is for this case.

Reply:
Done.